templates/core/build.html

```diff
@@ -199,6 +199,7 @@ <h2><span class="u-number-circle">2</span> What version of Ubuntu do you want?</
     <div class="col-10 u-sv3">
       <h2><span class="u-number-circle">3</span> What packages do you want preinstalled?</h2>
       <p>Select the applications that will be pre-installed on the device. For Ubuntu Core this will be snaps, for Ubuntu Classic it will be snaps or packages.</p>
+      <p>Your selected archtecture is <strong class="js-architecture"></strong>. If you are missing a snap its probably not supported by this archtecture.</p>
```

Comment:
Can this have a fallback value in case JS fails?

Reply:
Yeah sure, but the whole page is greyed out if no JS loads anyway so though the masking would cover it.

Reply:
Done, made the notification hidden until there is data.